src/nls/cs/strings.js

```diff
@@ -410,9 +422,11 @@ define({
     "CMD_TWITTER"                         : "{TWITTER_NAME} - Twitter",
     "CMD_ABOUT"                           : "O aplikaci {APP_TITLE}",
     "CMD_OPEN_PREFERENCES"                : "Otevřít soubor s preferencemi",
+    "CMD_OPEN_KEYMAP"                     : "Otevřít uživatelské klávesové zkratky",
 
     // Řetězce pro main-view.html
     "EXPERIMENTAL_BUILD"                   : "experimentální verze",
+    "RELEASE_BUILD"                        : "verze",
```

Comment:
"Stavět" is a verb meaning "to construct" or "to build" (building, structure). There are two possibilities, how to translate "build" in computer development domain. The first is "verze" and the second is "sestavení". I don't think, that the second possibility fits better. Trust me, I'm from the Czech republic. And btw, Google Translator is not good translator if you want to check czech language. You should use Lingea if you want to check translations